pkg/apis/datadoghq/v1alpha1/datadogagentdeployment_types.go

```diff
@@ -114,6 +114,14 @@ type DatadogAgentDeploymentSpecAgentSpec struct {
 	// SystemProbe configuration
 	// +optional
 	SystemProbe SystemProbeSpec `json:"systemProbe,omitempty"`
+
+	// Confd configuration
```

Comment:
could you add more documentation / context here based on https://docs.datadoghq.com/agent/guide/agent-configuration-files 🙏